### PR TITLE
Added example for create StatefulSet from file

### LINKED ISSUE #740: Added example for create StatefulSet from YAML file
There is a PatchExample.java for create a Deployment, but it does not uses yaml from file.
I am not sure to write a working file on `examples/`, but I think this will be a simple explain.

### PR DESCRIPTION
**Pull request by seongwon-kang**
Closes #740

### CODE REVIEW

File: examples/src/main/java/io/kubernetes/client/examples/YamlExample.java [hunk 1]
```diff
@@ -67,5 +75,46 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
...
             .build();
     System.out.println(Yaml.dump(svc));
+
+    // Read yaml configuration file, and deploy it
+    ApiClient client = Config.defaultClient();
+    Configuration.setDefaultApiClient(client);
+
+    // Example yaml file can be found in $REPO_DIR/test-sts.yaml
+    File file = new File("test-sts.yaml");
+    V1StatefulSet sts;
```
Comment (yue9944882) [inline examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: i suggest use service as example for a conjunctive context

File: examples/src/main/java/io/kubernetes/client/examples/YamlExample.java [hunk 2]
```diff
@@ -67,5 +75,46 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
...
+
+    sts = (V1StatefulSet) Yaml.load(file);
```
Comment (yue9944882) [inline examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: merge line L85-87

File: examples/src/main/java/io/kubernetes/client/examples/YamlExample.java [hunk 3]
```diff
@@ -67,5 +75,46 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
...
+            "default",
+            null,
+            new V1DeleteOptions(),
+            null,
+            null,
+            null,
+            null);
+    System.out.println(deleteResult);
+
+    System.exit(0);
```
Comment (yue9944882) [inline examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: remove this line

File: examples/src/main/java/io/kubernetes/client/examples/YamlExample.java [hunk 4]
```diff
@@ -67,5 +75,46 @@ public static void main(String[] args) throws IOException, ApiException, ClassNo
...
+  }
+
+  public static void setYamlKindMetadataForTest() {
```
Comment (yue9944882) [inline examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: inline this, no need for a new method

Comment (seongwon-kang) [reply examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: thank for your review, @yue9944882

File: examples/src/main/java/io/kubernetes/client/examples/YamlExample.java (hunk 3 above)
Comment (seongwon-kang) [reply examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: Throws `IllegalThreadStateException` if not exit, so #603 suggests `System.exit(0);` for incomplete. anyway, I'll try to remove it.

File: README.md [hunk 5]
```diff
@@ -76,6 +76,8 @@ We prepared a few examples for common use-cases which are shown below:
   Patch resource objects in various supported patch formats, equal to `kubectl patch`.
   - [FluentExample](https://github.com/kubernetes-client/java/blob/master/examples/src/main/java/io/kubernetes/client/examples/FluentExample.java): 
   Construct arbitrary resource in a fluent builder style.
+  - [YamlExample](https://github.com/kubernetes-client/java/blob/master/examples/src/main/java/io/kubernetes/client/examples/YamlExample.java): 
+  Create Yaml Configuration from Builder or file, and deploy to cluster.
```
Comment (yue9944882) [inline README.md]:
```suggestion
  Suggested way to load or dump resource in Yaml.
```

File: examples/src/main/java/io/kubernetes/client/examples/YamlExample.java (hunk 3 above)
Comment (yue9944882) [reply examples/src/main/java/io/kubernetes/client/examples/YamlExample.java]: mmmm it doesnt make a difference AFAIK.. i guess?

**Pull request merged**